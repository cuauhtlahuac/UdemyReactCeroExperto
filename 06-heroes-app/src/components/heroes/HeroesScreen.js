import React from 'react'

const HeroesScreen = () => {
  return (
    <div>
      <h1>Heroes</h1>
    </div>
  )
}

export default HeroesScreen
