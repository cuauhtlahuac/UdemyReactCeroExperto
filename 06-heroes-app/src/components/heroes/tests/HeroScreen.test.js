import React from 'react';
import { mount } from 'enzyme';
import HeroesScreen from '../HeroScreen';
import { MemoryRouter } from 'react-router-dom';

describe('Tests <HeroScreen />', () => {
	const historyMock = {
		length: 10,
		push: jest.fn(),
		goBack: jest.fn(),
	};

	const component = mount(
		<MemoryRouter>
			<HeroesScreen history={historyMock} />
		</MemoryRouter>,
	);

	test('should render correctly', () => {
		expect(component).toMatchSnapshot();
	});
});
