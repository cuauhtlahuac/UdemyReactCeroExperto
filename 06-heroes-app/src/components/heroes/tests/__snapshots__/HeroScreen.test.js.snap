// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests <HeroScreen /> should render correctly 1`] = `""`;
